main.js

```diff
@@ -134,13 +147,39 @@ function extractData() {
 	};
 }
 
+function getCleanGradesFromExams(exams) {
+	var clean = []
+	var dirty_exams = exams.get()
+	var found = 0;
```

Comment:
```suggestion
	var found = false;
```
Also, move it to inside the outer loop.